packages/webpack/__e2e-fixtures__/babel-postcss-extract-text/webpack.config.js

```diff
@@ -14,7 +14,7 @@ const postcss = require('@webpack-blocks/postcss')
 const extractText = require('@webpack-blocks/extract-text')
 const { css } = require('@webpack-blocks/assets')
 const path = require('path')
-const precss = require('precss')
+const postcssVars = require('postcss-simple-vars')
```

Comment:
`precss` is outdated and brings lots of dependencies with itself, but its features are not required for these tests.